src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/camunda/notification/CreateClaimRespondentNotificationHandler.java

```diff
@@ -53,9 +57,12 @@ public String camundaActivityId() {
 
     private CallbackResponse notifyRespondentSolicitorForClaimIssue(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        var recipient = callbackParams.getRequest().getEventId().endsWith("_CC")
```

Comment:
Should we use enum here to check if the event is equal to `NOTIFY_RESPONDENT_SOLICITOR1_FOR_CLAIM_ISSUE_CC`? it will be less error prone if we rename the event in future.

Reply:
good point, I'll update it in all places 👍